Tests/configure_and_test_integration_instances.py

```diff
@@ -1622,18 +1726,19 @@ def main():
     if build.is_nightly:
         build.install_nightly_pack()
     else:
-        pack_ids = get_non_added_packs_ids(build)
-        build.install_packs(pack_ids=pack_ids)
-
-        new_integrations, modified_integrations = build.get_changed_integrations()
-        pre_update_configuration_results = build.configure_and_test_integrations_pre_update(new_integrations,
-                                                                                            modified_integrations)
+        modified_packs_names = get_non_added_packs_ids(build)
+        packs_not_to_install = get_packs_not_to_install(modified_packs_names, build)
+        packs_to_install = modified_packs_names - packs_not_to_install
+        build.install_packs(pack_ids=packs_to_install)
+        new_integrations_names, modified_integrations_names = build.get_changed_integrations(packs_not_to_install)
```

Comment:
https://github.com/demisto/content/blob/master/Tests/private_build/configure_and_test_integration_instances_private.py#L123

Also, make sure to test the change E2E (run a build of a PR with this use case and make sure we don't install the unhidden pack in the pre-update part)

Reply:
Already did in the first 3 commits ([fails here](https://code.pan.run/xsoar/content/-/jobs/13852488), [succeed here](https://code.pan.run/xsoar/content/-/pipelines/3117456))

Reply:
what about the link I sent?